### ISSUE TITLE
Make the "Stream funds by the second" counter on the landing page update once per second

### ISSUE THREAD

**Issue by CodeSandwich** [drips-network/app#724]
The current infinitely fast counter doesn't match the `Drips enables token streams (...) by the second` description above. IMO it'd be a better illustration if it "ticked" only once per second.

Yeah, it's a nitpick, feel free to just close it.

**Comment by efstajas** [comment]
Estimates update currently every 100ms across the entire app, mostly just because it looks neat. 😁 You _can_ reduce the tick rate to 1s in settings, but it's not the default. I don't think the technicality of the funds actually only moving once per second vs. being represented as a more continuous stream matters much in practice. And IMO it's best to be consistent on the LP & app, so we'd have to make it update once per seconds globally.

Closing this for now as won't fix, but if anyone feels strongly about this, feel free to jump in 👍
